tests/common.go

```diff
@@ -295,6 +295,23 @@ func CollectSupport() {
 	})
 }
 
+// CheckForCoreFiles check if cores files are present on each node
+func CheckForCoreFiles() {
```

Comment:
Instead of `CheckForCoreFiles`, call this `PerformSystemCheck`.

Checking of core files is just an implementation detail for the node driver. At a high level here, we are just doing a system check.